React/Views/RCTScrollView.m

```diff
@@ -345,6 +345,12 @@ - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
   __block UIView *hitView;
 
   [_stickyHeaderIndices enumerateIndexesWithOptions:0 usingBlock:^(NSUInteger idx, BOOL *stop) {
+    NSUInteger subviewCount = [self contentView].reactSubviews.count;
```

Comment:
Can the subview count change during iteration? if not you might want to hoist this declaration outside of the block.